addons/hr_holidays/hr_dashboard.xml

```diff
@@ -38,7 +38,8 @@
 
                 <xpath expr="//div[contains(@class, 'o_kanban_primary_right')]" position="after">
                     <div class="col-xs-12 o_kanban_primary_bottom">
-                        <field name="absence_of_today" widget="progress" options="{'title': 'Absence', 'current_value': 'absence_of_today', 'max_value': 'total_employee', 'editable': false}"/>
+                        <a name="%(hr.act_employee_from_department)d" type="action" context="{'search_default_is_absent_totay':1}" >Absent Employees</a>
```

Comment:
there is already "hr_employee_action_from_department" action to show absent employee. no need to set context..